lib/model.js

```diff
@@ -676,18 +676,14 @@ class Model {
    * @return {Model}
    */
   static init(attributes, options) { // testhint options:none
-    options = options || {};
```

Comment:
FWIW, this is functionally the same since `options = {}`, fails the `options.sequelize` check at L681 (precommit, below), but I'll revert it.